language/tools/move-build/tests/test_runner.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use move_build::source_package::manifest_parser as MP;
+use move_command_line_common::testing::{format_diff, read_env_update_baseline, EXP_EXT};
+use std::{fs, path::Path};
+
+pub fn run_test(path: &Path) -> datatest_stable::Result<()> {
+    let update_baseline = read_env_update_baseline();
+    let exp_path = path.with_extension(EXP_EXT);
+
+    let exp_exists = exp_path.is_file();
+
+    let contents = fs::read_to_string(path)?;
+    let output = match MP::parse_move_manifest_string(contents).and_then(MP::parse_source_manifest)
+    {
+        Ok(parsed_package) => format!("{:#?}", parsed_package),
+        Err(error) => format!("{:#}", error),
```

Comment:
sticking a `\n` at the end of the output here should do the trick

Reply:
They should all have snazzy newlines now :)